sass/styles.scss

```diff
@@ -1 +1,547 @@
-p {color:red;}
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700|Roboto:100,300,400,500,700');
+
+/*variables*/
+$color-accent: #cc071e;
+$color-body: #353535;
+$body-font: Roboto, sans-serif;
+
+// $link: #000000;
+// $visit: #cc071e;
+// $hover: #cc071e;
+// $active: #000000;
+
+/*mixins*/
+@mixin link($link, $hover, $active) {
+  a {
+    text-decoration:none;
+    color: $link;
+      &:hover {
+        color: $hover;
+        text-decoration:underline;
+      }
+      &:active {
+        color: $active;
+      }
+  }
+}
+
+$html-margin: 0px auto !default;
+$padding: 0px !default;
+$font-size: 62.5% !default;
+
+
+@include link(black, #cc071e,black);
+
+
+body {
+    font-family: $body-font;
+    font-size:1.2em;
+    margin:0px;
+    padding:0px;
+}
+
+
+
+.detail-list {
+  padding-left:20px;
+    li {
+      list-style-position:inside;
+      list-style-type:circle;
+      font-weight:300;
+      margin-left:20px;
+  }
+}
+
+.detail-small {
+    width:100px;
+    margin-bottom:50px;
+}
+
+.features {
+    margin-bottom:70px;
+}
+
+form {
+  margin-top:60px;
+    p {
+      float:left;
+      display:block;
+      height:25px;
+  }
+  &:after {
+   content: "";
+   display: table;
+   clear: both;
+ }
+}
+
+html {
+    margin:0px auto;
+    width:320px;
+    padding:0px;
+    font-size:62.5%;
+}
+
+header {
+  margin-top:10px;
+    img {
+      height:70px;
+      margin-left:90px;
+      margin-bottom:30px;
+    }
+}
+
+h1 {
+    display:none;
+}
+
+h2 {
+    display:inline;
+    font-size:2.5rem;
+    text-transform:uppercase;
+    }
+
+h3 {
+    font-size:2.0em;
+    letter-spacing:0.5px;
+}
+
+img {
+    max-width:100%;
+}
+//
+// input[type="submit"] {
+//     font-size:0;
+//     width:120px;
+//     height:25px;
+//     border: none;
+//     display:block;
+// }
+
+li {
+    list-style:none;
+}
+
+nav ul {
+    display:none;
+}
+
+p {
+    color: $color-body;
+    font-weight:300;
+    line-height:147%;
+}
+
+.product-description h3 {
+    font-size:1.6rem;
+    font-weight:300;
+    padding-left:40px;
+}
+.products {
+  img {
+      margin-bottom:10px;
+      display:block;
+  }
+
+  li {
+      padding-right:10px;
+      padding-left:10px;
+      width:143px;
+      height:150px;
+      float:left;
+      box-sizing:border-box;
+      margin-bottom:20px;
+  }
+
+  .products ul {
+      text-align:center;
+      padding:0px;
+  }
+
+}
+
+.product-name {
+    font-size:1.5rem;
+    display:block;
+    margin-bottom:5px;
+}
+
+.product-price {
+    font-size:1.5rem;
+    font-weight:700;
+    letter-spacing:0.9px;
+}
+
+#quantity {
+    width:40px;
+    height:20px;
+    margin-right:22px;
+}
+
+.review {
+  width:320px;
+  display:block;
+  min-height:200px;
```

Comment:
While this `min-height` helps to enforce a grid perhaps it leads to undesired extra space in the shorter review. Consider removing this.

Reply:
Looking again the problem may instead by the `h3` you've nested in the first item. I'll make a comment on that in the markup.